services/galley/src/Galley/Run.hs

```diff
@@ -67,11 +68,13 @@ import Wire.API.Routes.Public.Galley
 import Wire.API.Routes.Version
 import Wire.API.Routes.Version.Wai
 import Wire.OpenTelemetry (withTracerC)
+import Wire.PostgresMigrations (runAllMigrations)
 
 run :: Opts -> IO ()
 run opts = lowerCodensity do
   tracer <- withTracerC
   (app, env) <- mkApp opts
+  lift $ runAllMigrations env._hasqlPool env._applog
```

Comment:
I'm wondering if this is necessary? What's the benefit of running the migrations in `galley` as well?

Reply:
This way galley wouldn't fail even if brig is taking its time to come up.

Reply:
Ah, I see :+1: